### ISSUE TITLE
Create JS console helper (create every leader content type) to Test push notifications 

### ISSUE THREAD

**Issue by petervojtek** [PowerlineApp/powerline-mobile#221]
@jterps08 wrote that following actions should induce native smartphone notification:

1) Create new leader content (all kinds) [this will also help us test leader content loading/responses]
2) Request to follow me
3) @Mention me
4) Invite me to a group you're in but I'm not in. 
5) Somehow boost a post in a group (may need Igor's help with this one)
6) Comment on a post I authored
7) Upvote a post I authored
8) Comment on a post you authored, but I subscribed to
9) Create a new post and a new petition after I've followed you (and you've approved).
10) I think that's all...

**Comment by petervojtek** [comment]
ok, I began with scenario 6): 
1. As Pavlik Morozov, on smartphone, I created a Post (http://localhost:8100/#/post/263)
2. as Peter12, I commented the post
3. as Pavlik Morozov, I expected to receive a smartphone push notification for that comment, but no push notification arrived. (regular social activity newsfeed notification arrived properly).

@jterps08 , please confirm the steps above really should trigger the push notification.

UPDATE 2016-09-12 - THE text below is no longer relevant, currently push notifications work for me:

@igaponov, can you please inspect that the push notification is fired from backend?
 this is what i see from smartphone regarding push notifications:
- I am sure that [push notification plugin](https://github.com/phonegap/phonegap-plugin-push) is contained in mobile app. 
- When I initialize the smartphone app, [I listed](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/EXAMPLES.md) to push notification registration and I receive the registration ID:

```
registrationId=ftQLENUtIMo:APA91bFfnXUgLJYtmZejxo4dtU5urwcvcPx7vI5EbfaBCh3WRStXu8_2NhyAS8JfITznf9d3a5FGuYVedYqT6pKvCZEMI3VOCT8So16ViI60B1_f89-J7Cp7neyVCtzb6WGfMwjQzVH_
```

then I listed to notifications, but none seems to arrive.

I use this SENDER_ID:

```
  <plugin name="phonegap-plugin-push" spec="1.6.0">
    <variable name="SENDER_ID" value="354736666363"/>
  </plugin>
```

I also tried it on emulator to make sure it is not my phone android security settings which prevent displaying notifications.

**Comment by jterps08** [comment]
In step 1, anyone who follows Pavlik should receive push notification alert. 
In step 2, Pavlik (the author of the post that Peter12 is commenting on) should receive a push notification alert by default for any new comments or upvotes (unless turned off). Anyone who opted to subscribe to that post could also receive a push notification when Peter12 comments.
In step 3, correct.

**Comment by igaponov** [comment]
Fix works only for new comments

**Comment by jterps08** [comment]
FYI: I tried creating new leader petition and new leader event in group Build Test Two (group 61). Items were created and published successfully and included as priority zone. However, no push notification was received. 

I do not have the 'cheat' code to create leader fundraisers, polls, news/discussion, or announcements.

**Comment by igaponov** [comment]
Bug with leader petition and leader event is fixed.

**Comment by petervojtek** [comment]
> I do not have the 'cheat' code to create leader fundraisers, polls, news/discussion, or announcements.

yes, I have to implement this in the JS console helper. i am blocked due to broken API doc now.

**Comment by petervojtek** [comment]
@jterps08 , what it Poll of `type:representative` ? should I implement the JS console helper for it?

[api](https://api-dev.powerli.ne/api-doc#post--api-v2-groups-{group}-polls) allows to create such poll type. the [glossary](https://github.com/PowerlineApp/powerline-server/wiki/Glossary) does not seem be explain what it is.

**Comment by jterps08** [comment]
Honestly, I'm not sure if I know what it is either... :-/ Skip it for now.

**For context:**

A long time ago we used to be able to create a poll inside a group (e.g. Save the Whales) but allow the group to share the poll results to a registered elected leader. I think this may be old code from that. If so, ignore.

OR

It may be when a representative is creating a poll (I don't think it's this). See https://github.com/PowerlineApp/powerline-server/issues/244 for more details. If it's this, skip for now and we'll come back when we tackle 244.

**Comment by igaponov** [comment]
@jterps08 
There are 4 types of `user` entity on backend:
- User
- Group
- Representative
- Superuser
  That means you can register as any of these types and login to app. There are also two different leader content types, for groups and representatives. We definitely want to remove this functionality for groups, because now user can manage his groups from app without logout. I don't know anything about other types.

**Comment by petervojtek** [comment]
@jterps08, I update the JS helper to allow create additional Poll:\* types:

https://github.com/PowerlineApp/powerline-mobile/wiki/HOWTO-Create-leader-content-from-JS-console 

which data type is there missing? I am ware of Discussion (not there yet).

**Comment by jterps08** [comment]
@petervojtek new announcement is the only thing missing. It's not a poll type, but it would be helpful in terms of leader content creation helper.

**Comment by petervojtek** [comment]
for announcement we are blocked by #246

**Comment by petervojtek** [comment]
ok, we have it all.
